runners/google-cloud-dataflow-java/src/main/java/org/apache/beam/runners/dataflow/DataflowRunner.java

```diff
@@ -2033,6 +2034,16 @@ private static void translate(
         PubsubUnboundedSink overriddenTransform,
         StepTranslationContext stepContext,
         PCollection input) {
+      if (overriddenTransform.getPublishBatchWithOrderingKey()) {
+        throw new UnsupportedOperationException(
+            String.format(
+                "%s does not currently support publishing with ordering keys. "
```

Comment:
nit: maybe better as "The Dataflow runner does not "  instead of the class name which won't mean much to a user since it is automatically inserted.

Reply:
Done